lib/lexer_mdx.mll

```diff
@@ -16,17 +16,21 @@ let labels l =
     failwith msg
 }
 
-let eol = '\n' | eof
-let ws = ' ' | '\t'
+let eol = '\n' | '\r' '\n' | eof
+let ws = [' ' '\t']
+
+let until_eol = [^'\n' '\r']
```

Comment:
Shouldn't this be called `non_eol` ? There is no quantifier of any kind so `until` is a bit missleading.

I think we should either move the `*` here or rename it.

Same applies to the other to `until_x` rules below!

Reply:
Indeed. I named it like this because I intended to write it like this: `([^'\n' '\r']* as x) eol` but that's not allowed (bits of regex can't return a value)